App.tsx

```diff
@@ -9,18 +9,24 @@ import { Header } from "./components/Header";
 import { TodoList } from "./components/TodoList";
 import { TodoContext, newTodoStore } from "./stores/Todo.store";
 import { NewDialog } from "./components/CreateEditDialog";
+import { useFonts } from "expo-font";
 
 const { tamaguiConfig } = initializeLibraries();
 
 const ThemedApp = observer(function ThemedApp() {
   const [store] = useState(newTodoStore);
-  if (!store.isLoaded) {
+  const [loaded] = useFonts({
```

Comment:
for cleaner code you could export this hook to its own file.

Reply:
Eh, splitting it to it's own file would just obfuscate what's going on imo